### PR TITLE
Fix typo from fmt.Sprint to fmt.Sprintf

### PR DESCRIPTION
**Pull request by gouyang**
Signed-off-by: Guohua Ouyang <gouyang@redhat.com>

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically.
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

**Comment by gouyang** [comment]
The conflict is solved, PTAL @shiywang.

File: tests/util.go [hunk 1]
```diff
@@ -53,6 +53,16 @@ func CreateResourceWithFilePathTestNamespace(filePath string) {
 	exec(Result{verb: "create", filePath: filePath, nameSpace: NamespaceTestDefault})
 }
 
+func createResourceWithFilePath(resourceType, resourceName, filePath, nameSpace string) {
```
Comment (shiywang) [inline tests/util.go]: This function has already removed, we don't need it anymore.

**Comment by gouyang** [comment]
The problem does not exists any more.

**Pull request closed without merging**